src/OrchardCore.Modules/OrchardCore.Forms/Models/TextAreaPart.cs

```diff
@@ -5,6 +5,13 @@ namespace OrchardCore.Forms.Models
     public class TextAreaPart : ContentPart
     {
         public string DefaultValue { get; set; }
+
         public string Placeholder { get; set; }
+
+        public LabelOptions LabelOption { get; set; }
+
+        public string Label { get; set; }
+
+        public ValidationOptions ValidationOption { get; set; }
     }
```

Comment:
remove extra lines

Reply:
In other PRs I was asked to add lines between property name. It seems to be the new style

Reply:
I in this PR one place you have no lines and other place you add extra lines.  If you believe extra lines should be there then add them to other classes. or remove from here